packages/react-core/src/helpers/ouia.ts

```diff
@@ -19,6 +19,30 @@ export function getOUIAProps(componentType: string, id: OuiaId | undefined, ouia
   return {
     'data-ouia-component-type': `PF4/${componentType}`,
     'data-ouia-safe': ouiaSafe,
-    'data-ouia-component-id': id === undefined ? uid++ : id
+    'data-ouia-component-id': id === undefined ? `OUIA-${componentType}-${++uid}` : id
   };
 }
+
+const ouiaIdByRoute: any = {};
+/**
+ * Returns a generated id based on the URL location
+ *
+ * @param {string} componentType OUIA component type
+ * @param {string} variant Optional variant to add to the default
+ */
+export function getDefaultOUIAId(componentType: string, variant?: string) {
```

Comment:
Can this take a 3rd prop (say, arguments?) that gets hashed instead of a global counter?

I fear a global counter will increment when unmount/remounting components (like everything inside a Drawer's content).

Do you expect remounted components to contain new generated `OUIAId`s @RonnyPfannschmidt / @quarckster / @psav ?

Reply:
As we want correct used ids never to be generated, we shouldn't mind a little remount churn, as that helps to ensure people see them as as fragile as they are